src/Uno.UI/UI/Xaml/Controls/PasswordBox/PasswordBox.cs

```diff
@@ -17,6 +17,9 @@ public partial class PasswordBox : TextBox
 		private readonly SerialDisposable _revealButtonSubscription = new SerialDisposable();
 
 		public PasswordBox()
+#if __MACOS__
+			:  base(true)
```

Comment:
Looks like this constructor in TextBox is not used at all, we can probably remove it completely.

Reply:
Removed and added only on .macOS partial